docs/source/conf.py

```diff
@@ -87,7 +87,7 @@
 smv_branch_whitelist = multiversion_regex_builder(BRANCHES)
 # Defines which version is considered to be the latest stable version.
 # Must be listed in smv_tag_whitelist or smv_branch_whitelist.
-smv_latest_version = 'v1.15'
```

Comment:
nit: since docs are multiversion, configuration file is taken from master, backporting this isn't needed.